esx_service/cli/vmdkops_admin.py

```diff
@@ -284,10 +290,6 @@ def commands():
                                     'help': 'Maximum size of the volume that can be created',
                                     'metavar': 'Num{MB,GB,TB} - e.g. 2TB'
                                 },
-                                '--volume-maxcount': {
-                                    'help': 
-                                    'Maximum number of volumes to create on the datastore for this tenant'
-                                },
                                 '--volume-totalsize': {
```

Comment:
Nit, not your change, but should this be like --max_limit as its a resource (capacity) entitlement for the tenant on the datastore and also --max_volume_size?